README.rst

```diff
@@ -118,13 +118,40 @@ DNS lookup...
 * If ``gethostbyname`` does not have it cached nor in the ``hosts`` file then a request
   is made to the known DNS server that was given to the network stack. This is typically the
   local router or the ISP's caching DNS server.
+  OnWindows it will use an algorithm to determine which DNS server to try first
+  to resolve the hostname. http://technet.microsoft.com/en-us/library/dd197552(WS.10).aspx
 * The local DNS server (or local gateway's) MAC address is looked up in the ARP
   cache. If the MAC address is missing, an ARP request packet is sent.
 * Port 53 is opened to send a UDP request to DNS server (if the response size is
-  too large, TCP will be used instead).
+  too large, TCP will be used instead, but it's definitely not the case for a simple A or AAAA
+  request to www.google.com).
 * If the local/ISP DNS server does not have it, then a recursive search is
   requested and that flows up the list of DNS servers until the SOA is reached,
   and if found an answer is returned.
+* Almost every time, this DNS servers is not resolving the google.com (the only exceptions are
```

Comment:
This sounds funny, maybe you mean 'this DNS server is not resolving to google.com'?